jwst/regtest/test_nirspec_mos_spec3.py

```diff
@@ -6,21 +6,28 @@
 from jwst.stpipe import Step
 from stdatamodels.jwst import datamodels
 
+# Mark all tests in this module
+pytestmark = [pytest.mark.bigdata]
+
 
 @pytest.fixture(scope="module")
-def run_pipeline(rtdata_module):
+def run_pipeline(rtdata_module, request_tracker):
```

Comment:
```suggestion
def run_pipeline(rtdata_module, resource_tracker):
```
What's `request_tracker`?